### PR TITLE
refine message wording

### PR DESCRIPTION
**Pull request by herohua**

### CODE REVIEW

File: src/credential/credentialController.ts [hunk 1]
```diff
@@ -70,10 +70,10 @@ export class CredentialController {
             this.eventStream.post(new UserSignInTriggered(correlationId));
             let userInfo;
             if(this.environmentController.docsRepoType === 'GitHub') {
-                this.eventStream.post(new UserSignInProgress(`Sign-in to docs build with GitHub account...`, 'Sign-in'));
+                this.eventStream.post(new UserSignInProgress(`Signed in to Docs with GitHub account...`, 'Sign-in'));
```
Comment (928PJY) [inline src/credential/credentialController.ts]: >Signed [](start = 62, length = 6)

Signing in #Closed

File: src/observers/buildStatusBarObserver.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export class BuildStatusBarObserver extends BaseStatusBarObserver {
                 this.setAndShowStatusBar(`$(sync~spin)`, undefined, undefined, 'Building the current workspace folder');
                 break;
             case EventType.BuildInstantReleased:
-                this.setAndShowStatusBar(`$(sync)`, undefined, undefined, 'Ready to Build');
+                this.setAndShowStatusBar(`$(sync)`, undefined, undefined, 'Ready to build');
```
Comment (928PJY) [inline src/observers/buildStatusBarObserver.ts]: >build [](start = 84, length = 5)

validate

Comment (herohua) [reply src/observers/buildStatusBarObserver.ts]: I am kind of struggling between `build` and `validate` - would you agree at this moment `validate` is a better choice across the board?

Comment (928PJY) [reply src/observers/buildStatusBarObserver.ts]: Yes, confirmed with Curt before, he also agree to use `validate`

---
In reply to: [393566233](https://github.com/docascode/vscode-docs-build/pull/98#discussion_r393566233) [](ancestors = 393566233)

**Review by 928PJY — APPROVED** [review]
:shipit:

**Pull request merged**